src/webhook_app.py

```diff
@@ -3,46 +3,27 @@
 """
 import flask
 import flask.logging
-from flask import Flask, Response, request, render_template
+from   flask import Flask, Response, request
 import glob
 import hmac
 import logging
 import os
 import re
 import subprocess
-from subprocess import PIPE
+from   subprocess import PIPE
```

Comment:
Errant whitespace change

```suggestion
from subprocess import PIPE
```